.env

```diff
@@ -1,3 +0,0 @@
-NEXT_PUBLIC_BASE_API_URL=https://survey-api.nimblehq.co/api/v1/
-NEXT_PUBLIC_AUTH_CLIENT_ID=pV3EjKhvaQ7gA_-TcU9wO6h0clj3igOxpLPGuZ3JkA8
-NEXT_PUBLIC_AUTH_CLIENT_SECRET=K9rM_oUarNIJi6BwZm42cDeIV7wVQyumDqVHB0joTRQ
```

Comment:
If the sole purpose of deleting this file from version control is not to let anyone abuse the credentials, then what do you think of changing(editing) the credentials in the oauth2 application site? Because these changes are present in the git history and if anyone wants to abuse, they can still do it!

Reply:
Yes, I revoke and create new tokens